pkg/minikube/cluster/types.go

```diff
@@ -40,6 +40,7 @@ type KubernetesConfig struct {
 	KubernetesVersion string
 	NodeIP            string
 	APIServerName     string
+	DNSDomain         string
```

Comment:
Ah I forgot that the issue was that the flag already existed but there was no way to configure it.